cmd/util/doc-gen/generators/traitmetadatagen.go

```diff
@@ -113,6 +130,7 @@ func (g *traitMetaDataGen) Finalize(c *generator.Context, w io.Writer) error {
 			return err
 		}
 
+		fmt.Fprintf(file, "%s", string(licenseHeader))
```

Comment:
Is it this required? I mean, in the doc we are not adding licenses in general, so, if we're doing here we may need to add to the rest of the doc as well?

Reply:
If you run the make check-licenses the pkg/resources/traits.yaml is listed as missing license. So this is just for him. For the doc we should be fine.